internal/cmd/kafka/command_topic_consume.go

```diff
@@ -34,13 +37,6 @@ func newConsumeCommand(prerunner pcmd.PreRunner, clientId string) *cobra.Command
 		),
 	}
 
-	c := &hasAPIKeyTopicCommand{
-		HasAPIKeyCLICommand: pcmd.NewHasAPIKeyCLICommand(cmd, prerunner),
```

Comment:
We lose some warnings about not using `--environment` and `--cluster` flags. We might want to add those warnings elsewhere if we don't want to use this command struct.